src/asm/parse.rs

```diff
@@ -132,7 +132,10 @@ pub enum Result {
 }
 
 /// Parses the assembly function at `path` from the file `file`.
-#[cfg_attr(feature = "cargo-clippy", allow(use_debug, cyclomatic_complexity))]
+#[cfg_attr(
+    feature = "cargo-clippy",
+    allow(clippy::use_debug, clippy::cognitive_complexity)
+)]
```

Comment:
This should just be `#[allow(clippy:: ... )]`

Reply:
Alright, wasn't sure whether this would break anything in subtle ways.